tests/build.rs

```diff
@@ -985,6 +985,15 @@ fn crate_authors_env_vars() {
                 execs().with_status(0));
 }
 
+// The tester may already have LD_LIBRARY_PATH=::/foo/bar which leads to a false positive error
+fn setenv_for_removing_empty_component() {
+    let search_path = env::var(dylib_path_envvar()).expect("var()");
+    let new_search_path =
+        env::join_paths(env::split_paths(&search_path).filter(|e| !e.as_os_str().is_empty()))
+            .expect("join_paths");
+    env::set_var(dylib_path_envvar(), new_search_path);
+}
```

Comment:
Wondering if it really makes sense to modify the framework code just for an minor issue like this... But points taken, it isn't nice to modify the environment of the test driver.